### DISCUSSION TITLE
error trying to install moc on mac

### DISCUSSION THREAD

**Discussion by scorpio32** [Homebrew/discussions#426] [Everyday usage]
trying to install moc on mac i get : 
Error: libbluray: Calling depends_on :java is disabled! Use "depends_on "openjdk@11", "depends_on "openjdk@8" or "depends_on "openjdk" instead.
Please report this issue to the homebrew/core tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/libbluray.rb:23

**Comment by carlocab** [comment]
What does `brew config` and `brew doctor` say?

**Reply by scorpio32** [reply]
brew config reads: 
HOMEBREW_VERSION: 2.7.0-102-gbcbf585
ORIGIN: https://github.com/Homebrew/brew
HEAD: bcbf5857dac003eeb6506c32721de74d7c8222ac
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a864ded16ee555c333cd99b7056e213eda5796f4
Core tap last commit: 8 months ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: N/A
Git: 2.26.2 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.6-x86_64
CLT: N/A
Xcode: N/A

**Comment by carlocab** [comment]
Your core tap is very much outdated.
```
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a864ded16ee555c333cd99b7056e213eda5796f4
Core tap last commit: 8 months ago
```
Do `brew update` and try again. If it doesn't work, please paste the output of `brew update` here.
